### ISSUE TITLE
Resume page

### ISSUE THREAD

**Issue by justinlee0777** [justinlee0777/ecommerce-site#5]
Description:
Contingent on #2's completion.
Create this page of the site: https://alexayang03.wixsite.com/vassarportfolio/resumé .

Acceptance criteria:
- Page anchor is out of scope. Will update this issue with issue for anchor.
- Masonry is out of scope. Will update this issue with created issue for masonry.

First iteration:
- Overview and Educational History sections
